test/base/SpecBase.scala

```diff
@@ -105,7 +105,7 @@ trait SpecBase
     new GuiceApplicationBuilder()
       .overrides(
         bind[DataRequiredAction].to[DataRequiredActionImpl],
-        bind[IdentifierAction].to[FakeIdentifierAction],
+        bind[IdentifierAction].to[FakeOrgIdentifierAction],
```

Comment:
Have we got duplicate actions here? 
FakeOrgIdentifierAction and FakeIdentifyOrgAction 😗

Reply:
On the surface they are the same, however whenever I make the change it doesn't seem to find the type FakeIdentifyOrgAction :(